tests/cross_functional/ui/test_creation_and_deletion_of_sc_and_rbdpool.py

```
@@ -88,29 +91,34 @@ def test_new_sc_new_rbd_pool(
             volume_binding_mode=volume_binding_mode,
         )
 
-        log.info(f"Creating a PVC using {sc_obj.name}")
+        logger.info(f"Creating a PVC using {sc_obj.name}")
         pvc_obj = pvc_factory(
             interface=interface_type, storageclass=sc_obj, size=10, status=pvc_status
         )
-        log.info(f"PVC: {pvc_obj.name} created successfully using " f"{sc_obj.name}")
+        logger.info(f"PVC: {pvc_obj.name} created successfully using " f"{sc_obj.name}")
```

Ruff (0.15.15)

[warning] Implicitly concatenated string literals on one line

Combine string literals

(ISC001)